typescript/web/src/components/auth-manager/signin-modal/signin-modal.tsx

```diff
@@ -161,8 +167,25 @@ export const SigninModal = ({
                     </>
                   ) : (
                     <>
-                      <FormControl id="email">
+                      <FormControl
+                        id="email"
+                        isRequired
+                        isInvalid={isEmailInvalid}
+                      >
                         <Input
+                          onChange={(event) => {
+                            setIsEmailInvalid(
+                              !validateEmail(event.target.value)
```

Comment:
Maybe here we should use this:

```
!validateEmail(event.target.value) || event.target.value.length === 0
```

This poses a problem because when the user clicks on the form and it is empty it has a grey border. Then the user types something and then erases it, the border is red until the input is blurred. We therefore have here two different states of the border color for the same state of the input value, which can be confusing

Reply:
I am not sure that the change you propose here does what you want.

Basically, you want to always make the border red when the length is 0, right?
`onChange` isn't called until you type the first character. Therefore, when clicking on the input, it will be blue until you type something. Then if you erase all its content it will become red. 

I think the best workflow would be:
- onFocus, make it blue if it contains a valid email, else, if invalid or empty => red
- onChange, valid => blue; invalid or empty => red
- onBlur, invalid => remains red, valid or empty => default state (grey)

Whatt do you think?

Reply:
I implemented this behaviour in 578d9a11101cc610d74009425dd2fe78470496e5, if you want to give it a try

Reply:
Changed discussed during he DSM have been done in aef5493a66000431fc052b9dd0e39fb8d4d5ed45